src/utils/types.ts

```diff
@@ -1,23 +1,18 @@
-import { ApiResponse } from 'apisauce';
-import { Error } from '~app/hooks/useRequest';
-
 export type Nullable<T> = T | null;
 
-interface InfoUser{
+export interface InfoUser {
+  id: number;
   email: string;
   firstName: string;
   lastName: string;
   locale: string;
 }
 
-export interface UserRegister extends InfoUser{
+export interface UserRegister extends InfoUser {
```

Comment:
i think thiis is not used anymore

Reply:
is used in the registration form:
`useForm<UserRegister>`